x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/index.ts

```diff
@@ -27,6 +27,7 @@ import { getLeadingBreadcrumbsForSecurityPage } from './get_breadcrumbs_for_page
 import type { GetSecuritySolutionUrl } from '../../link_to';
 import { useGetSecuritySolutionUrl } from '../../link_to';
 import { useIsGroupedNavigationEnabled } from '../helpers';
+import * as i18n from '../../../../detection_engine/rule_details_ui/pages/rule_details/translations';
```

Comment:
I think importing an i18n constant from this domain-specific path to common components is an indication that something is wrong with the dependencies.